app/uk/gov/hmrc/cataloguefrontend/connector/model/model.scala

```diff
@@ -23,7 +23,9 @@ import play.api.mvc.{PathBindable, QueryStringBindable}
 
 case class Username(value: String) extends AnyVal
 
-case class TeamName(asString: String) extends AnyVal
+case class TeamName(asString: String) extends AnyVal {
+  override def toString: String = asString
```

Comment:
Can we keep `toString` as the default. For logging, debugging, reading test failures etc. it's useful to see the type. This also aligns with all our other Stringable value classes.